openrndr-core/src/main/kotlin/org/openrndr/shape/Shape.kt

```diff
@@ -200,23 +201,29 @@ class Shape(val contours: List<ShapeContour>) {
         return contours.hashCode()
     }
 
+    /** Applies a boolean union operation between two [Shape]s. */
     fun union(other: Shape): Shape = union(this, other)
+
+    /** Applies a boolean difference operation between two [Shape]s. */
     fun difference(other: Shape): Shape = difference(this, other)
+
+    /** Applies a boolean intersection operation between two [Shape]s. */
     fun intersection(other: Shape): Shape = intersection(this, other)
 
+    /** Calculates a [List] of all points of where paths intersect between two [Shape]s. */
```

Comment:
I don't know about the "of where" expression... could be my English. Mentioned earlier, repeated in the next few functions.